snapcraft/plugins/catkin.py

```diff
@@ -245,12 +246,18 @@ def build(self):
 
         self.run(
             ['rm', '-f',
-             'opt/ros/' + self.options.rosversion + '/.catkin',
-             'opt/ros/' + self.options.rosversion + '/.rosinstall',
-             'opt/ros/' + self.options.rosversion + '/setup.sh',
-             'opt/ros/' + self.options.rosversion + '/_setup_util.py'],
+             'opt/ros/' + self.options.rosversion + '/.rosinstall'],
             cwd=self.installdir)
 
+        # Fix the shebang in _setup_util.py to be portable
```

Comment:
I agree, but ALL the build steps here need test coverage, which I plan on adding in a later PR.